scripts/travisci/check_precommit.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
+export PYTHONPATH=.
```

Comment:
isn't this done in the docker entrypoint?

Reply:
It 's not set elsewhere but i found `conda activate garage` does the thing so we are safe to remove it.